src/algorithms.rs

```diff
@@ -655,9 +692,9 @@ fn div_rem_core(mut a: BigUint, b: &BigUint) -> (BigUint, BigUint) {
         let mut prod = b * &q0;
 
         while cmp_slice(&prod.data[..], &a.data[j..]) == Greater {
-            let one: BigUint = One::one();
-            q0 = q0 - one;
-            prod = prod - b;
+            let one: BigDigit = 1;
+            q0 -= one;
```

Comment:
nit: combine this line with the previous

Reply:
The small advantage here is in the type ascription, but actually `1u32` should work just as well for arithmetic, even when `BigDigit` is `u64`. I'll try it.